guides/common/modules/proc_restricting-custom-repositories-to-a-specific-os-version-or-architecture.adoc

```diff
@@ -0,0 +1,20 @@
+[id="Restricting_Custom_Repositories_to_a_Specific_OS_Version_{context}"]
+= Restricting Custom Repositories to a Specific Operating System or Architecture in {Project}
+
+You can configure {Project} to make custom repositories available only on hosts with a specific operating system version or architecture.
+For example, you can restrict custom repositories only to {RHEL}{nbsp}9 hosts.
+
+ifdef::satellite[]
+[NOTE]
+Only restrict architecture and operating system version for custom products.
+{Project} applies these restrictions automatically for Red{nbsp}Hat repositories.
+endif::[]
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Click the product that contains the repository sets you want to restrict.
+. In the *Repositories* tab, click the repository you want to restrict.
```

Comment:
```suggestion
. In the *Repositories* tab, select the repository you want to restrict.
```

Reply:
In the web UI, there is a checkbox next to the name of the repository and then there is the (clickable) name of the repository. 
* If we use _select_, it would mean selecting the checkbox.
* I want to use _click_ to make it clear that the user needs to click the name of the repository.

I've seen "select" used in foreman-docs instead of "click", and I understand the need for consistency, but it's actually misleading in this case.

From the IBM style guide:
* click verb: Use click [...] to refer to the action of selecting something on a graphical interface.
* select verb: Use select to indicate a mouse action that applies to both mobile and non-mobile devices when a term such as click might create confusion.

Reply:
Thanks for looking it up!